tests/spec/fs.writeFile-readFile.spec.js

```diff
@@ -22,6 +22,19 @@ describe('fs.writeFile, fs.readFile', function() {
     });
   });
 
+  
+  it('should error when path is wrong to writeFile',function(done){
+    var fs = util.fs();
+
+    fs.writeFile('/tmp/myfile', '','utf8', function(error, result) {
+      expect(error).to.exist;
+      expect(error.code).to.equal('ENOENT');
+      expect(result).not.to.exist;
+      done();
+    });
+
```

Comment:
We can also try use Promises to write the test, for instance:
it('(promise) should return an error when path is wrong to writeFile', ()=>{
var fsPromises = util.fs().promises;
return fsPromises.writeFile (......)
.catch(error => { expect...... }); // and you don't need done() at the end.
});